testcases/two_zone_apartment_hydronic/models/library_versions.json

```diff
@@ -0,0 +1,5 @@
+{
+"IBPSA_COMMIT": "8a0d2372925b93c5c5aa536eb3afceee3a07bcb3",
+"BUILDINGS_COMMIT": "69bb7cf6b12898a2b17ad9d9371ed0eec3faaa12",
+"IDEAS_COMMIT": "32860eadbf59bee955993da936a8c5aeac2d90df"
+}
```

Comment:
Please use version numbers if these commits refer to releases of the library.  Best is to stick to releases of the libraries if possible.

Reply:
new `library_version.json` is:

`{"BUILDINGS_COMMIT": "v8.1.3",
"IDEAS_COMMIT": "v2.2.1"`}